src/Security/Member.php

```diff
@@ -1691,7 +1691,7 @@ protected function encryptPassword()
 
         // Overwrite the Password property with the hashed value
         $this->Password = $encryption_details['password'];
-        $this->Salt = $encryption_details['salt'];
+        $this->Salt = $encryption_details['salt'] ?: '';
```

Comment:
Why is `salt` being explicitly set to a falsy value?

Reply:
Not sure, best guess is there was some unit test that used to pass because `false` wasn't being validated, though safetly set to either false or null in the database

Reply:
Revert